nevergrad/parametrization/core.py

```diff
@@ -241,6 +241,17 @@ def __repr__(self) -> str:
 
     # %% Constraint management
 
+    def _constraint_penalties(self) -> float:
+        """Returns the list of constraint penalties penalties
```

Comment:
please update the docstring, and probably the name if you do not want to return the list of penalities.
I still think the list of penalties is better since it provides more flexibility though (eg: multiobjective)

Actually, returning a list of floats would be much better since this be used in the `satisfies_constraints` method just below, and avoid duplication, so it's probably the way to go, and you can just sum in the `_constraint_solver` function.

Reply:
Deleted as you suggested.